client/src/Try/App.purs

```diff
@@ -0,0 +1,70 @@
+module Try.App
+  ( AppT(..)
+  , Error(..)
+  , ParAppT
+  , runAppT
+  ) where
+
+import Prelude
+
+import Control.Monad.Error.Class (class MonadThrow)
+import Control.Monad.Except (ExceptT(..))
+import Control.Monad.Except as ExceptT
+import Control.Parallel as Parallel
+import Control.Parallel.Class (class Parallel)
+import Data.Array.NonEmpty (NonEmptyArray)
+import Data.Either as Either
+import Data.Functor.Compose (Compose(..))
+import Data.Newtype as Newtype
+import Data.Validation.Semigroup (V)
+import Data.Validation.Semigroup as V
+import Effect.Aff.Class (class MonadAff)
+import Effect.Class (class MonadEffect)
+
+data Error
+  = FetchError String
+  | FFIErrors (NonEmptyArray String)
+
+instance semigroupError :: Semigroup Error where
+  append (FetchError err) _ = FetchError err
+  append _ (FetchError err) = FetchError err
```

Comment:
I _think_ this preserves the current behavior for non-FFI errors, since we were using `ExceptT` which short-circuits on first error.